trade-finance-manager-api/src/v1/emails/AIN-MIN-confirmation/gef-facilities-list.test.js

```diff
@@ -82,20 +85,60 @@ describe('generate AIN/MIN confirmation email facilities list email variable/str
 
       expect(result).toEqual(expected);
     });
+
+    it('should format isUsingFacilityEndDate to Yes when true', () => {
+      const result = facilityFieldsObj({ isUsingFacilityEndDate: true });
+
+      expect(result).toEqual({ isUsingFacilityEndDate: 'Yes' });
+    });
+
+    it('should format isUsingFacilityEndDate to No when false', () => {
+      const result = facilityFieldsObj({ isUsingFacilityEndDate: false });
+
+      expect(result).toEqual({ isUsingFacilityEndDate: 'No' });
+    });
```

Comment:
I think instead of these tests we can simply update line 73 in this test to call/assert with `mapBooleanToYesOrNo`, since the function is consuming this rather than having yes/no logic

Reply:
I've had a quick go at this but found myself having to mock the `gef-facilities-list` module, ensuring that the `facilityFieldsObj` is not mocked. Think just checking the functionality here is much simpler